wlutil/build.py

```diff
@@ -44,14 +44,19 @@ def handleHostInit(config):
 
        run([config['host-init'].path] + config['host-init'].args, cwd=config['workdir'])
 
-def handleBinPost(config):
+def handlePostBin(config, linuxSrc, linuxBin):
```

Comment:
linuxSrc is config['linux-src'] and linuxBin is config['bin']. I can't think of a scenario where that wouldn't be true so you don't really need separate args for it.

Reply:
I think the `linux-src` is fine to get rid from the arg list. However, `linux-bin` can either be a "no-disk" or a "normal" version so just using `config['bin']` is not sufficient (unless Im mistaken). So when I invoke the function I give it the direct path to the "right" binary i.e. the "-nodisk" or the normal path.

Reply:
alright, that's fine.